test/fixtures/dialects/ansi/select_from_values.sql

```diff
@@ -0,0 +1,5 @@
+select * from values 1, 2;
```

Comment:
readers may want to skip ahead to the [working cases](https://github.com/sqlfluff/sqlfluff/pull/2245/files#diff-65b86a0a7aee1334a536d1ee3a75631989ca84ca93754cccffbe9421c03f34f1R1)

these are still all split up from my troubleshooting

is there a way to document known statements that are legal but do not parse, like marking them as expected failures? or you do all that through issues?

Reply:
Yes, I would probably create issues for those.